src/components/SidePanel/LoadSave.jsx

```diff
@@ -47,6 +48,7 @@ import { triggerState } from '../../reducers/triggerReducer';
 
 export const Load = () => {
     const dispatch = useDispatch();
+    useHotKey(['command+o', 'ctrl-o'], () => dispatch(load()));
```

Comment:
Are these hotkeys disabled when the component itself is disabled?

Reply:
Yes, when the Load button is not rendered (because a device is selected or we are on the About pane) Cmd/Ctrl-O does nothing. Similar for the Save shortcut.

Reply:
This is not true, since disabled state is evaluated a few lines later.

Reply:
The ctrl hotkeys are wrong btw, it should be + not -, so currently it does not work on Windows

Reply:
Also I tested the hotkeys while saving should be disabled and this hotkey overrides that like Bence suspected. Good catch Bence!

Reply:
You are right, I did not think about the case that the Save button is rendered but disabled. For now this is not worth it, I just reverted that commit again in c9753bd.